packages/frontend-shared/src/locales/en-US.json

```diff
@@ -650,9 +650,20 @@
       "switchTestingTypeInfo": "There is {n} {testingType} test failing in this spec. To run it locally switch to {testingType} testing. | There are {n} {testingType} tests failing in this spec. To run them locally switch to {testingType} testing.",
       "switchTestingTypeAction": "Switch to {testingType} testing"
     },
-    "notLoggedIn": "Not logged in",
-    "notConnected": "Not connected",
-    "noRuns": "No runs",
+    "overPlanLimit": "Over plan limit",
+    "reachedMonthlyUsage": "You've reached the monthly usage limit for your billing plan.",
+    "upgradeYourPlan": "Upgrade your plan to continue recording runs.",
+    "upgradePlan": "Upgrade plan",
+    "contactAdmin": "Contact admin",
+    "wellDone": "Well Done!",
+    "allYourTestsPassed": "All your tests passed.",
```

Comment:
I feel like this should be "All _of_ your tests passed.", but I could be overthinking it

Reply:
yeah I had this thought as well... 🤔 just went based off of what the mockup said. According to [some blog](https://proofed.com/writing-tips/word-choice-all-of/), we could use either for this sentence

Reply:
leaving this for now